resync/api/middleware/cors_monitoring.py

```
@@ -201,77 +129,38 @@ def monitor_cors() -> CORSMonitor:
-                logger.error("exception_caught", error=str(e), exc_info=True)
-                self.cors_monitor.log_violation(
-                    origin,
-                    request.url.path,
-                    f"Malformed origin URL: {str(e)}",
-                    request.method,
-                    request.headers.get("access-control-request-headers", ""),
-                )
-
-        # Continue with the request
-        return await self.app(scope, receive, send_wrapper)
+                logger.error('exception_caught', exc_info=True, extra={'error': str(e)})
```

Ruff (0.15.1)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)